xml/System.Net.Http/StreamContent.xml

```diff
@@ -66,7 +66,14 @@
       <Docs>
         <param name="content">The content used to initialize the <see cref="T:System.Net.Http.StreamContent" />.</param>
         <summary>Creates a new instance of the <see cref="T:System.Net.Http.StreamContent" /> class.</summary>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ The <xref:System.Net.Http.StreamContent> object calls <xref:System.IO.Stream.Dispose> on the provided <xref:System.IO.Stream> object when <xref:System.Net.Http.StreamContent.Dispose%2A?displayProperty=nameWithType> is called.  
+  
```

Comment:
The formatting on these should be fixed.... Whatever ports these comments in adds double spaces to the ends of the line, indents the starting paragraph for no reason, and doesn't properly space the headers. All of these should be fixed before merging.

@carlossanlop FYI

```suggestion
  
## Remarks

The <xref:System.Net.Http.StreamContent> object calls <xref:System.IO.Stream.Dispose> on the provided <xref:System.IO.Stream> object when <xref:System.Net.Http.StreamContent.Dispose%2A?displayProperty=nameWithType> is called.

```